test/math/flightMath.spec.js

```diff
@@ -26,11 +26,11 @@ ava('.calcTurnRadius() returns a turn radius based on speed and bank angle', t =
     t.true(result === expectedResult);
 });
 
-ava('.calculateHeadingFromTwoPositions()', t => {
+ava('.bearingToPoint()', t => {
```

Comment:
I won't knock you on the lack of description here because, well, I was naughty and didn't have one there first.  In the future, though, we should have descriptions for our tests.  I'm very guilty here, but something to keep in mind.